---
float32[6] curpos
